services/metadata/backend/data/db.json

```diff
@@ -1,9 +1,50 @@
 {
     "projects": [
-        {"id": "0000"},
-        {"id": "9998"},
-        {"id": "9997"},
-        {"id": "082D"}
+        {
+            "id": "0000",
+            "name": "Testprojektname (test)",
+            "description": "Dies ist ein Testprojekt...alle Properties wurden verwendet, um diese zu testen"
```

Comment:
I see why you added this to do the preview. I'm just wondering (might be wrong though) if it would be more realistic if that wasn't available here, and the preview would need to fetch this data from the metadata route too?

Reply:
Yeah, just to make life easier for now ;)